src/style.css

```
@@ -1 +1,67 @@
 /* Styles go here */
+html {
+  font-family: Roboto, Helvetica, sans-serif;
+  font-size: 12px;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  box-sizing: border-box;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+/* .nav {
+  margin-top: -15px; 
+} */
+
+.nav__bar {
+  display: flex;
+  list-style: none;
+  flex-direction: row;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__elements:not(:last-child) {
```

That's not a good idea to make class name plural
```suggestion
.nav__element:not(:last-child) {
```